cli/command/container/exec.go

```diff
@@ -57,6 +58,7 @@ func NewExecCommand(dockerCli command.Cli) *cobra.Command {
 	flags.StringVarP(&options.user, "user", "u", "", "Username or UID (format: <name|uid>[:<group|gid>])")
 	flags.BoolVarP(&options.privileged, "privileged", "", false, "Give extended privileges to the command")
 	flags.VarP(&options.env, "env", "e", "Set environment variables")
+	flags.StringVarP(&options.workdir, "workdir", "w", "", "Working directory inside the container")
```

Comment:
Can you move this below the version annotation for `env`, and add a version annotation for the new option? The annotation is used to hide the flag when talking to an older daemon version (which doesn't yet support the feature)

```go
flags.SetAnnotation("env", "version", []string{"1.25"})
flags.StringVarP(&options.workdir, "workdir", "w", "", "Working directory inside the container")
flags.SetAnnotation("workdir", "version", []string{"1.35"})
```